### PR TITLE
Improve the WASM developer package interface

### PR DESCRIPTION
**Pull request by josephschorr**
This change modifies the interface and call convention used for invoking the development package via WASM to make everything more standardized on protocol buffers.

This change also introduces CI testing of the WASM interface and deprecates the V0 developer API, minus the portions still necessary for sharing.

### CODE REVIEW

File: pkg/development/wasm/operations_test.go [hunk 1]
```diff
@@ -0,0 +1,858 @@
...
+			},
+			nil,
+		},
+		{
+			"invalid namespace name",
+			`definition fo {}`,
+			[]*core.RelationTuple{},
+			tuple.MustParse("somenamespace:someobj#anotherrel@user:foo"),
+			&devinterface.DeveloperError{
+				Message: "error in object definition fo: invalid NamespaceDefinition.Name: value does not match regex pattern \"^([a-z][a-z0-9_]{1,62}[a-z0-9]/)?[a-z][a-z0-9_]{1,62}[a-z0-9]$\"",
```
[the remark this answers is not in the document]
Comment (josephschorr) [reply pkg/development/wasm/operations_test.go:57]: Yeah, we can fix that later

File: pkg/development/wasm/operations_test.go [hunk 2]
```diff
@@ -0,0 +1,858 @@
...
+			"{}\n",
+		},
+		{
+			"invalid validation yaml",
+			`definition somenamespace {}`,
+			[]*core.RelationTuple{},
+			`asdkjhgasd`,
+			"",
+			&devinterface.DeveloperError{
+				Message: "unexpected value `asdkjhg`",
```
[the remark this answers is not in the document]
Comment (josephschorr) [reply pkg/development/wasm/operations_test.go:269]: Its truncated by the YAML lib

**Comment by josephschorr** [comment]
Updated

**Review by samkim — APPROVED** [review]
LGTM

**Pull request merged**